requirements-dev.txt

```diff
@@ -1,8 +1,11 @@
+cython
 flake8
 flake8-print
 notebook
 pytest
 pytest-cov
+pytest-cython
 restructuredtext-lint
 sphinx
 sphinx_rtd_theme
+tqdm
```

Comment:
tqdm is a progress bar indicator. It is useful when calling HPO from command line. I don't want to dump too much debug info, but still give some useful measure of progress to the user. If there are python-inbuilt progress estimators, let me know. I will be happy to use those.